model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -98,6 +113,29 @@ public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
 
+    /**
+     * Gets the gender of this provider
+     *
+     * @return the current gender of this client
+     */
+    public Gender getGender(){
+        return gender;
+    }
+
+    /**
+     * Sets a new gender to this client
+     *
+     * @param gender an updated gender of this client
+     */
+    public void setGender(Gender gender){
+        if (!(Gender.FEMALE.equals(gender) || Gender.MALE.equals(gender) ||
+                Gender.OTHERS.equals(gender))){
+            throw new IllegalArgumentException("This is not a valid input. Please try \"MALE\"" +
+                    "or \"FEMALE\" or \"OTHERS\"");
+        }
```

Comment:
This is no longer necessary since the input is an enum, so it will always be one of those three values, or null. In either case, I don't think this check is still necessary.